cli/internal/run/run.go

```diff
@@ -829,6 +712,11 @@ func parseRunArgs(args []string, output cli.Ui) (*RunOptions, error) {
 
 	unresolvedCacheFolder := filepath.FromSlash("./node_modules/.cache/turbo")
 
+	// --scope and --since implies --include-dependencies for backwards compatibility
```

Comment:
should it be --scope _or_ --since?

Reply:
`--since`-only was previously handled here: https://github.com/vercel/turborepo/blob/main/cli/internal/run/run.go#L250
It does not pull in dependencies.

`--scope`-only was previously handled here: https://github.com/vercel/turborepo/blob/main/cli/internal/run/run.go#L253
It also does not pull in dependencies.

It was only when they were both specified, and there were actual changed packages, that dependencies were included: https://github.com/vercel/turborepo/blob/main/cli/internal/run/run.go#L238
This amounts to building all changed packages and their dependencies, regardless of specified scope. I don't think this behavior is correct, but I'm not sure if we want to stop building dependencies in that case since users might be relying on it? Although they are somewhat arbitrary...

Reply:
Right, what I'm saying is that I think in addition to these changes, passing either `--scope` or `--since` should automatically `--include-dependencies` and `--no-deps` flags (breaking behavior).

Reply:
Oh, gotcha. Yeah, we can do that.

Reply:
As mentioned here, we're going to keep the behavior in this PR, without expanding it to "`--scope` or `--since` triggers `--include-dependencies`. This closely mirrors existing behavior. We can then look at alternate ways of specifying packages.